src/app/admin/page.tsx

```
@@ -0,0 +1,40 @@
+    } catch (error) {
+      console.error('Error fetching team length:', error)
+    }
+  }
+
+  return (
+    <section className="flex-col-center h-full gap-4">
+      <button onClick={getHealthCheck} className={`${btnCommonStyle} bg-system-green`}>
+        health check
+      </button>
+      <button onClick={getApiTest} className={`${btnCommonStyle} bg-system-blue`}>
+        api test
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)